src/test/java/edu/tamu/cap/auth/service/AppUserCredentialsServiceTest.java

```diff
@@ -51,7 +51,7 @@ public final class AppUserCredentialsServiceTest {
 
     @BeforeEach
     public void setup() throws JsonParseException, JsonMappingException, IOException {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
We shouldn't be using either of these:
```
        MockitoAnnotations.initMocks(this);
        MockitoAnnotations.openMocks(this);
```

instead, prefix the class with this annotation:
```
@ExtendWith(MockitoExtension.class)
```

Also, the `@SpringBootTest` annotation already includes `@ExtendWith(SpringExtension.class)`, so please remove this from line ~31.